sdk/keyvault/azure-keyvault-keys/azure/keyvault/keys/crypto/_models.py

```diff
@@ -54,24 +59,33 @@ def __init__(self, key_id, algorithm, signature):
 class VerifyResult:
     """The result of a verify operation.
 
-    :param bool result: Whether the signature is valid
+    :param str key_id: The encryption key's Key Vault identifier
+    :param bool is_valid: Whether the signature is valid
+    :param algorithm: The encryption algorithm used
```

Comment:
~encryption~ signature

Reply:
switched this and the type of the algorithm, thanks!